modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElixirClientCodegen.java

```diff
@@ -665,6 +672,10 @@ public String decodedStruct() {
             } else if (isArray && languageSpecificPrimitives().contains(baseType)) {
                 return "[]";
             }
+
+            if (this.baseType.equals("AnyType")){
+                return "%{}";
```

Comment:
Why are we returning an empty map here? It doesn’t look *wrong*, just *odd*.

Reply:
You're absolutely right, this line is unnecessary since the if/else check above will already handle `AnyType` (since it's now marked as primitive) and this check will never be reached for AnyType. Nice catch :D!